ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/PurchaseButton.kt

```diff
@@ -183,3 +184,27 @@ private fun PurchaseButtonPreview() {
         )
     }
 }
+
+private fun TemplateConfiguration.PackageConfiguration.hasDifferentCallToActionText(): Boolean {
+    val firstText = with(all.first()) {
+        introEligibilityText(
+            introEligibility,
+            localization.callToAction,
+            localization.callToActionWithIntroOffer,
+            localization.callToActionWithMultipleIntroOffers,
+        )
+    }
+
+    all.forEach {
```

Comment:
Maybe skip the first?

Reply:
Done in https://www.npmjs.com/package/@revenuecat/purchases-capacitor#setloglevel